### PR TITLE
Bridge Headset Rearrangement

### PR DESCRIPTION
**Pull request by Karmic-Skink**
## About The Pull Request
This PR alters the encryption keys for both the XO and Bridge STAFF.
XO gets a captain key to better facilitate assisting the Captain.
Bridge Staff have had their superfluous channels removed.

## Why It's Good For The Game
Allows for better communication channels and removes non required channels/flags.
Fixes: #1210

## Changelog
:cl:
add: Added Bridge Staff encryption key
balance: Altered XO headset to use Captain's encryption key
remove: Removed Independent from MT's encryption key
/:cl:

### CODE REVIEW

**Comment by AU518987077** [comment]
Thank you, i and countless others will appreciate this change very strongly once this gets merged.

**Review by Jalleo — APPROVED** [review]
I approve. A second maintainer can merge this if they approve it too.

**Review by Kmc2000 — APPROVED** [review]
Yeah, I'm fine with this. XO deserves a bigger headset

**Pull request merged**
